src/main/java/types/Backup.java

```diff
@@ -103,4 +103,14 @@
      * @since 4.3
      */
     @Link Disk[] disks();
+
+    /**
+     * A reference to the snapshot created if the backup is of type hybrid.
```

Comment:
"is of type hybrid" is not very clear, maybe:

    is using a snapshot.

Or when we have a documented API for using/not using snapshot:

    was started using the with_snapshot option.